src/views/wait-for-others-to-sign.njk

```diff
@@ -11,30 +11,17 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <div class="interruption-card">
-          <h1 id="blue-interruption-card" class="govuk-heading-xl govuk-!-margin-bottom-1">You must ask the other directors or members to sign the application</h1>
+          <h1 id="blue-interruption-card" class="govuk-heading-xl govuk-!-margin-bottom-1">The directors or members must sign the application before you can submit it</h1>
```

Comment:
I checked on local env there, and the banner doesn't seem to style blue properly. Can you double-check this?

Reply:
I had this problem once, had to docker pull the newest cdn.

Reply:
Cool cool 👍